### PR TITLE
Add TestifySec to list of Friends

### PR DESCRIPTION
**Pull request by jkjell**
TestifySec is a huge fan of in-toto.  We'd like to add our name to the list of friends of the project.

### CODE REVIEW

File: testifysec/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# TestifySec
+
+TestifySec is a supply chain security company that has created two open source projects that leverage `in-toto`.
```
Comment (adityasaky) [inline testifysec/README.md:3]:
```suggestion
[TestifySec](https://www.testifysec.com/) is a software supply chain security company that has created two open source projects that leverage `in-toto`. 
```

File: testifysec/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# TestifySec
+
+TestifySec is a supply chain security company that has created two open source projects that leverage `in-toto`. 
+
+## Witness
+[Witness](https://github.com/testifysec/witness/#key-features) contains partial implementation of the in-toto specification including ITE-5, ITE-6, and ITE-7, and an embedded rego policy engine for building policy enforcement (which will align with ITE-10 in the future).
+
+## Archivista
+[Archivista](https://github.com/testifysec/archivista#archivista) is a graph and storage service for `in-toto` attestations. It allows the querying of metadata via a GraphQL API.
```
Comment (adityasaky) [inline testifysec/README.md:9]: nit: can you add a trailing new line?

**Review by adityasaky — APPROVED** [review]
LGTM minus some suggestions. Thanks!

**Review by JustinCappos — APPROVED** [review]
LGTM!

**Pull request merged**